Dockerfile

```
@@ -0,0 +1,30 @@
+FROM ubuntu
+
+LABEL version="0.1"
+LABEL description="A super docker image."
+
+RUN apt-get update
+
+RUN apt-get -y install curl git gnupg
+RUN curl -sL https://deb.nodesource.com/setup_12.x -O
+RUN bash setup_12.x
+
+RUN apt-get -y install nodejs
+
+RUN mkdir -p /home/app 
+
+WORKDIR /home/app
+
+ADD server.js server.js
+ADD index.html index.html
+ADD package.json package.json
+
+RUN apt-get install -y wget
+RUN wget https://www.lescinskas.lt/assets/img/posts/2019/docker.png -O background.png
+
+RUN npm install
+
+# Expose Port for the Application 
+EXPOSE 80 443
+
+CMD [ "node", "server.js" ]
```

You should consider launching your main command here using an Entrypoint. You can find more about it [here](https://docs.docker.com/build/building/best-practices/#entrypoint).

For example, you could create a script as the entrypoint to allow running a shell like bash for debugging purpose.

A really basic usage:
```suggestion
ENTRYPOINT [ "node" ]
CMD [ "server.js" ]
```